.pipelines/onebranch/pipeline.buildrp.official.yml

```diff
@@ -13,7 +13,8 @@ pr: none
 
 variables:
   Cdp_Definition_Build_Count: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
-  LinuxContainerImage: cdpxlinux.azurecr.io/user/aro/ubi8-gotoolset-1.17.7-13:20220526   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+  ONEBRANCH_AME_ACR_LOGIN: cdpxb8e9ef87cd634085ab141c637806568c00.azurecr.io
+  LinuxContainerImage: cdpxb8e9ef87cd634085ab141c637806568c00.azurecr.io/b8e9ef87-cd63-4085-ab14-1c637806568c/official/go-toolset-1.17.7-13:latest   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
```

Comment:
I forget if you can use a variable inside another variable in an ADO pipeline but it might be useful here.

Reply:
This pipeline definition extends a OneBranch template with strict rules. The `ONEBRANCH_AME_ACR_LOGIN` var is required here because our isolated ACR requires authentication (and cdpxlinux.azurecr.io didn't). Here's the documentation for this part: https://msazure.visualstudio.com/AzureRedHatOpenShift/_wiki/wikis/ARO.wiki/189194/CDPx-Container-Images hopefully this helps add context.

Reply:
Yes, that is what I was trying to get at.

Reply:
@dem4gus @SudoBrendan thanks for the suggestion, I added it in and it appears to be working: https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=58826293&view=logs&j=dcf80fb5-4153-5d36-7b0a-1b867e82342a